### ISSUE TITLE
haplotype inference for local region

### ISSUE THREAD

**Issue by Shicheng-Guo** [odelaneau/shapeit4#64]
Hi Olivier, 

I want to use shapeit4 for a small genomic region phasing (300Kbp). Is there something I should be careful for such task? for example the minimum genome requirement, or parameter setting? 

Thanks 

Shicheng

**Comment by odelaneau** [comment]
Hi,

What you should do is : (i) phase the full dataset and (ii) extract the region of interest from the phased dataset.

What you should not do: (i) extract the region of interest and (ii) phase it.

In the second approach you just remove some data that could inform the phasing of your region of interest.

Hope that helps

Best,

Olivier.

**Comment by Shicheng-Guo** [comment]
Hi Olivier, Thank you so much for your suggestion. I can understand that the approach 1 is the best strategy. However, I am working on 150K UKB-WGS data which makes the phasing process quite time-consuming, that's why I am trying to use approach 2 to save time. 

I am wondering do you think it will be okay with flanking 1Mbp region to have enough accuracy? 

Thanks

Shicheng

**Comment by odelaneau** [comment]
Yes, definitely, +/-1Mbp on each side is clearly enough to retain good accuracy.

Best,

Olivier
